src/theme/themeTest/ThemeTest.tsx

```diff
@@ -0,0 +1,36 @@
+import { ThemeProvider } from "@mui/material";
+import Box from "@mui/material/Box";
+import Stack from "@mui/material/Stack";
+import { Theme } from "@mui/material/styles";
+import React from "react";
+
+import { oldLightMaterialUITheme } from "../theme";
+import { ButtonTest } from "./buttons/Button";
+import { ButtonGroupTest } from "./buttons/ButtonGroup";
+import { IconButtonTest } from "./buttons/IconButton";
+import { SwitchTest } from "./buttons/Switch";
+import { ToggleButtonTest } from "./buttons/ToggleButton";
+import { AutocompleteTest } from "./inputs/Autocomplete";
+import { SelectTest } from "./inputs/Select";
+import { TextFieldTest } from "./inputs/TextField";
+import { TypographyTest } from "./typography/Typography";
+
+export default function ThemeTest({ theme = oldLightMaterialUITheme }: { theme?: Theme }) {
```

Comment:
👍
We can even create a theme playground based on this page sometime later. It would be great if we could adjust the component's properties and see how the component behaves based on these changes